<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8" />
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
        <meta name="description" content="" />
        <meta name="author" content="" />
        <title>Cas Blaauw</title>
        <!-- <link rel="icon" type="image/x-icon" href="assets/img/favicon.ico" /> -->
        <!-- Font Awesome icons (free version)-->
        <script src="https://use.fontawesome.com/releases/v5.15.3/js/all.js" crossorigin="anonymous"></script>
        <!-- Google fonts-->
        <link href="https://fonts.googleapis.com/css?family=Saira+Extra+Condensed:500,700" rel="stylesheet" type="text/css" />
        <link href="https://fonts.googleapis.com/css?family=Muli:400,400i,800,800i" rel="stylesheet" type="text/css" />
        <!-- Core theme CSS (includes Bootstrap)-->
        <link href="css/styles.css" rel="stylesheet" />
    </head>
    <body id="page-top">
        <!-- Navigation-->
        <nav class="navbar navbar-expand-lg navbar-dark bg-primary fixed-top" id="sideNav">
            <a class="navbar-brand js-scroll-trigger" href="#page-top">
                <span class="d-block d-lg-none">Cas Blaauw</span>
                <span class="d-none d-lg-block"><img class="img-fluid img-profile rounded-circle mx-auto mb-2" src="assets/img/profile.jpg" alt="..." /></span>
            </a>
            <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation"><span class="navbar-toggler-icon"></span></button>
            <div class="collapse navbar-collapse" id="navbarResponsive">
                <ul class="navbar-nav">
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#intro">Introduction</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#projects">Projects</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#skills">Skills</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#about-me">About me</a></li>
<!--                     <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#awards">Awards</a></li> -->
                </ul>
            </div>
        </nav>
        <!-- Page Content-->
        <div class="container-fluid p-0">
            <!-- About-->
            <section class="resume-section" id="intro">
                <div class="resume-section-content">
                    <h1 class="mb-0">
                        Cas<span style="opacity:0.15">per H.</span> 
                        <span class="text-primary">Blaauw</span>
                    </h1>
                    <div class="subheading mb-5">
                        Studying MSc Bioinformatics @ UCPH · Bioinformatics Student Assistant @ Clinical Microbiomics · Intern @ Winther group & Joshi group
                    </div>
                    <p class="lead">I'm a bioinformatician, currently studying the <a href="https://studies.ku.dk/masters/bioinformatics/" target="_blank"> MSc Bioinformatics at the University of Copenhagen</a> 
                        and working as a part-time bioinformatician at <a href="https://clinical-microbiomics.com" target="_blank">Clinical Microbiomics</a>. 
                        Besides this, I am a research intern working with <a href="https://scholar.google.com/citations?user=MMYgK1oAAAAJ&hl" target="_blank">Hiren Joshi</a> and <a href="https://olewinther.github.io/" target="_blank">Ole Winther</a> 
                        and I'm the president of <a href="http://cbiovikings.org/" target="_blank">CBioVikings, the Danish regional student group</a> of the <a href="http://www.iscb.org/" target="_blank">ISCB</a>.</p>
                    </p>
                    <p class="lead">
                        My main interest is understanding <b>how systems come together to form a greater whole</b>. These can range from subway systems to online communities, 
                        but my primary focus is <b>understanding the rules behind the molecular mechanisms of life</b>, whether you call it -omics, ML, systems biology, or something else.
                        I went into Molecular Life Sciences at Utrecht University hoping to study this in the lab, but realised my questions were best answered behind a computer instead.
                        The rest of my four years there were dedicated to biology, programming and statistics (including a minor in Applied Data Science), 
                        with a detour to the University of Hong Kong to study public policy and international relations. Alongside this, I worked in a variety of roles in education (from tutor to substitute teacher) for those four years.
                    </p>
                    <p class="lead mb-5">
                        Besides bioinformatics, I care about properly understanding and visualising data, science and technology's impact on society, and (inter)national public policy.
                        In my free time, some of my passions are baking, cities, design, languages, and obscure geographical knowledge.
                        To learn more about me, see my CV <a href="https://www.linkedin.com/in/cas-blaauw/" target="_blank">on LinkedIn</a>, explore my code <a href="https://github.com/CasBlaauw" target="_blank">on GitHub</a>, or scroll down for more details about my <a href='#projects'>research</a> and <a href='#interests'>motivation</a>.
                    </p>
                    <div class="social-icons">
                        <a class="social-icon" href="https://www.linkedin.com/in/cas-blaauw/" target="_blank"><i class="fab fa-linkedin-in"></i></a>
                        <a class="social-icon" href="https://github.com/CasBlaauw" target="_blank"><i class="fab fa-github"></i></a>
                        <a class="social-icon" href="https://twitter.com/CasBlaauw" target="_blank"><i class="fab fa-twitter"></i></a>
                        <a class="social-icon" href="mailto:casblaauw@gmail.com" target="_blank"><i class="fas fa-envelope"></i></a>
                    </div>
                </div>
            </section>
            <hr class="m-0" />
            <!-- projects-->
            <section class="resume-section" id="projects">
                <div class="resume-section-content">
                    <h2 class="mb-5">Projects</h2>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Machine learning for microbiome analysis</h3>
                            <div class="subheading mb-2">Clinical Microbiomics</div>
                            <p>Built a ML pipeline to robustly link microbiome abundance with various phenotypes.</p>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">2021</span></div>
                    </div>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Language modeling of proteins for glycosylation site prediction</h3>
                            <div class="subheading mb-2"><a href="https://olewinther.github.io/" style="color:inherit" target="_blank">University of Copenhagen</a></div>
                            <p>Used BERT-based language models, trained on UniProt rather than Wikipedia, to distill protein sequences to their core grammatical context in the language of life. 
                                Armed with the contextual representations for a variety of glycosylated sequences, I set out to figure out the rules behind O-GalNAc glycosylation, a biological process under complex multi-level regulation. </p>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">2021</span></div>
                    </div>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Automating and standardising report creation</h3>
                            <div class="subheading mb-2">Clinical Microbiomics</div>
                            <p>Built a system to automate the production of production-level reports, integrating data and report text in RMarkdowns and exporting to fully-styled Microsoft Office documents.</p>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">2021</span></div>
                    </div>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Dynamic eQTL mapping in development and stress</h3>
                            <div class="subheading mb-2"><a href="https://ubc.uu.nl/user/bsnoek/" style="color:inherit" target="_blank">Utrecht University</a></div>
                            <p>Investigated the dynamics of transcriptional regulation during cellular stress and development.
                                Incorporating this transcriptomic data with eQTL mapping provided new insights in the dynamics of transcriptomic regulation and increased eQTL detection accuracy.
                            </div>
                        <div class="flex-shrink-0"><span class="text-primary">2019-2020</span></div>
                    </div>
                    <div class="d-flex flex-column flex-md-row justify-content-between">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Homomerisation state prediction using symmetry-driven protein docking</h3>
                            <div class="subheading mb-2"><a href="https://www.bonvinlab.org/" style="color:inherit" target="_blank">Utrecht University</a></div>
                            <p>Built a proof-of-concept pipeline to predict protein homomer stoichiometry from monomer structures using HADDOCK, an information-driven protein docking software.</p>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">2018</span></div>
                    </div>
                </div>
            </section>
            <hr class="m-0" />
            <!-- Skills-->
            <section class="resume-section" id="skills">
                <div class="resume-section-content">
                    <h2 class="mb-4">Skills</h2>
                    <h3 class="mb-3">Competences</h3>
                    <ul class="fa-ul lead">
                        <li>
                            <span class="fa-li"><i class="fas fa-check"></i></span>
                            Data analysis
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-check"></i></span>
                            Statistical machine learning
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-check"></i></span>
                            Tool development
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-check"></i></span>
                            Data visualisation
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-check"></i></span>
                            Science pedagogy
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-check"></i></span>
                            Societal impact assessment
                        </li>
                    </ul>
                    <h3 class="mb-3">Programming Languages & Tools</h3>
                    <ul class="mb-0 lead">
                        <li>
                            <b>R</b> - [tidyverse | Shiny | officedown | mlr3 | DESeq2]
                        </li>
                        <li>
                            <b>Python</b> - [numpy | pandas | PyTorch | scikit-learn | biopython]
                        </li>
                        <li>
                            <b>Other tools</b> - [bash | git | LaTeX]
                        </li>
                    </ul>
                </div>
            </section>
            <hr class="m-0" />
            <!-- Interests-->
            <section class="resume-section" id="about-me">
                <div class="resume-section-content">
                    <h2 class="mb-5">About me</h2>
                    <p>[Under construction]
                    I always like to say that I ended up in life sciences due to <a href="https://gfycat.com/powerlessripeafricanjacana" target="_blank">a single gif</a>, 
                        which solidified my fascination for the molecular mechanisms of life. 
                    However, it had fierce competition from programmes like computer science, environmental science, human geography, and economics & policy 
                        when picking my bachelor's degree (which you do before enrolling in the Netherlands) and the systems and society approaches from those fields are something that I still try to apply within bioinformatics. 
                    I did really like biology and chemistry's experiment-based approach to science though, and so I enrolled in Molecular Life Sciences. 
                    The programme, primarily hosted by the chemistry department, did a great job in emphasising understanding over memorisation and research over rote learning. This mindset really shaped me as a budding scientist, 
                        even if I left the chemistry department behind after my first year. 
                        
                    Their research and teaching, although excellent, was focused on structural biology, and my dreams were bigger: understanding genomes was much more of my thing than painstakingly figuring out individual protein structures.
                    Instead, I indulged in programming and statistics, as deciphering (biological) datasets to understand large-scale biology was what really got my attention. 
                    In a way, that was a natural fit, as it led me back to the choice between computer science and molecular life science I had to make years prior, and even included the systems view I liked regardless of field. 
                    </p>
<!--                     <p class="mb-0">To stay engaged with 'the world' during my studies, I spent </p> -->
                    <p class="mb-0"> At the end of my Molecular Life Sciences bachelor, it was very clear I wanted to go fully into bioinformatics and leave the wetlab behind, especially as I'd been doing a minor in Applied Data Science and a computational thesis. 
                    Looking around Europe, I decided to go study in Copenhagen: perhaps the biggest life sciences hub in Europe, with great bioinformatics faculty hosting a great MSc programme, in a lovely city to boot.
                    Here, I have the opportunity to get acquainted with leading bioinformatics work in both academia and industry.</p>
                </div>
            </section>
            <hr class="m-0" />
            <!-- Awards-->
<!--             <section class="resume-section" id="awards"> -->
<!--                 <div class="resume-section-content"> -->
<!--                     <h2 class="mb-5">Something extra that's not this header</h2> -->
<!--                     <ul class="fa-ul mb-0"> -->
<!--                         <li> -->
<!--                             <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span> -->
<!--                             - Awarded a fellowship for the ICML Workshop in Computational Biology-->
<!--                         </li> -->
<!--                         <li> -->
<!--                             <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span> -->
<!--                             - -->
<!--                         </li> -->
<!--                     </ul> -->
<!--                 </div> -->
<!--             </section> -->
        </div>
        <!-- Bootstrap core JS-->
        <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"></script>
        <!-- Core theme JS-->
        <script src="js/scripts.js"></script>
    </body>
</html>
